lib/tasks/elasticsearch.rake

```diff
@@ -0,0 +1,9 @@
+namespace :elasticsearch do
+  desc 'Delete the "resource" index'
+  task :delete_resource_index do
+    client = Elasticsearch::Client.new(
```

Comment:
Is there a way to access the existing client instance without creating a new one?

Heroku will run take tasks in their own dynos so it doesn't matter, I'm more curious if it's possible.